Sources/Commands/SwiftBuildTool.swift

```diff
@@ -68,6 +120,39 @@ public class SwiftBuildTool: SwiftTool<BuildToolOptions> {
             print("warning: minimum recommended clang is version 3.6, otherwise you may encounter linker errors.")
         }
     }
+
+    private func buildOption(graph: PackageGraph) -> BuildOption? {
```

Comment:
Can you explain "should compute what we need to do using the provided build option"?